Tchap/Modules/Favourites/FavouriteMessages/FavouriteMessagesViewController.swift

```diff
@@ -0,0 +1,270 @@
+// File created from ScreenTemplate
+// $ createScreen.sh Favourites/FavouriteMessages FavouriteMessages
+/*
+ Copyright 2020 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import UIKit
+import Reusable
+
+final class FavouriteMessagesViewController: UIViewController {
+    
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let estimatedRowHeight: CGFloat = 21.0
+        static let contentInset = UIEdgeInsets(top: 10.0, left: 0.0, bottom: 10.0, right: 0.0)
+    }
+    
+    // MARK: - Properties
+    
+    // MARK: Outlets
+
+    @IBOutlet private weak var tableView: UITableView!
+    
+    // MARK: Private
+
+    private var viewModel: FavouriteMessagesViewModelType!
+    private var theme: Theme!
+    private var keyboardAvoider: KeyboardAvoider?
+    private var errorPresenter: MXKErrorPresentation!
+    private var activityPresenter: ActivityIndicatorPresenter!
+    private var isViewAppearedOnce: Bool = false
+    
+    private var roomBubbleCellDataList: [RoomBubbleCellData] = []
+
+    // MARK: - Setup
+    
+    class func instantiate(with viewModel: FavouriteMessagesViewModelType) -> FavouriteMessagesViewController {
+        let viewController = StoryboardScene.FavouriteMessagesViewController.initialScene.instantiate()
+        viewController.viewModel = viewModel
+        viewController.theme = ThemeService.shared().theme
```

Comment:
Unfortunately we failed for the moment to use Element theme in Tchap.
We only support 2 styles (related to a unique Theme)
Please replace the code related to the theme with the one related to a style (like in `AuthenticationViewController.swift` (or `RoomViewController.m`))

Reply:
I replaced Theme by Style